mod_reforged/hooks/entity/tactical/enemies/bandit_marksman.nut

```diff
@@ -68,7 +68,26 @@
 					return _weight;
 				}
 			})
-			if (armor != null) this.m.Items.equip(::new(armor));
+
+			if (armor != null)
+			{
+				this.m.Items.equip(::new(armor));
+
+				if (::Math.rand(1, 100) <= ::Reforged.Config.ArmorAttachmentChance.Tier3)
+				{
+					local armorAttachment = ::Reforged.ItemTable.ArmorAttachmentNorthern.roll({
+						Apply = function ( _script, _weight )
+						{
+							local conditionModifier = ::ItemTables.ItemInfoByScript[_script].ConditionModifier;
+							if (conditionModifier > 20) return 0.0;
+							return _weight;
+						}
+					})
+
+					if (armorAttachment != null)
+						this.getBodyItem().setUpgrade(::Reforged.new(armorAttachment));
```

Comment:
No need for `::Reforged.new` if you aren't modifying the instantiated item. Just use `::new` in this and other similar cases.